docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1040,6 +1041,29 @@
           on appropriate nodes.</para>
         </sect4>
       </sect3>
+
+      <sect3 id="ENV_values_yaml">
+        <title>Environment Values</title>
+
+        <para>You can define Environment Variables in a YAML file. The
+        environment values are defined under the
+        <emphasis>global.env:</emphasis> portion of the delivered values.yaml
+        as illustrated below.</para>
```

Comment:
as a list of name value pairs, as illustrated below.

Otherwise you haven't really described the format.